### ISSUE TITLE
JAWS narrate menu, when overriding role with aria-roledescription

### ISSUE THREAD

**Issue by kolaps33** [FreedomScientific/standards-support#286]
### Summary

1. Go to https://9snoy.csb.app/
2. Navigate into menu element "John Deer"
3. Observe JAWS narrattion 

### Expected result
1.
We would expect that JAWS doesn't narrate 'menu', when menu is override by aria-roledescription attribute. 

.2. 
If "participant" custom string which is in aria-roledescription attribute should be narrated on each element?, because when pure menu structure is used, then there is not "menuitem" role narrated. 

### Actual result
menu
John Deerparticipant 
1 of 3
To move through items press up or down arrow.

### Example
https://9snoy.csb.app/

#### JAWS version and build number
2019.1907.42

 
#### Operating System and version
Win 10 Enterprise
Version: 1903

#### Browser and version:
Chrome
Version 76.0.3809.132 (Official Build) (64-bit)

**Comment by JAWS-test** [comment]
Chrome and Furefox with JAWS 2019:

aria-roledescription is output at the end as a description. The role remains "menu". This seems to be so for some elements that they are not overwritten by aria-roledescription. But for many elements it works.

With role=menu you have a typo: "ariaroledescription" instead of "aria-roledescription". But this does not change the output.

**Comment by kolaps33** [comment]
Thank you for typo catch. Updated example with fix.

**Comment by JAWS-test** [comment]
two more remarks:
* first you should ask yourself which role is the right one (independent of roledescription), because all properties of the actual role are preserved. I can't tell you that, because I don't know which function is defined there.
* If aria-roledescription works correctly, it must of course name the role correctly, otherwise the user will not know how to operate the element. "participant" is more of a label than a role description. A role description would be e.g. "participant menu"

**Comment by JAWS-test** [comment]
https://rawgit.com/w3c/aria/master/#aria-roledescription: " Authors SHOULD limit use of aria-roledescription to clarifying the purpose of **non-interactive container roles** like group or region, or to providing a more specific description of a widget"

**Comment by stevefaulkner** [comment]
Closing this, please feel free to re-open if this still occurs with Latest (JAWS 2024).
